sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationClientBuilder.java

```diff
@@ -38,12 +36,14 @@
 import com.azure.core.util.builder.ClientBuilderUtil;
 import com.azure.core.util.logging.ClientLogger;
 import com.azure.core.util.serializer.JacksonAdapter;
-import java.net.MalformedURLException;
-import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import com.azure.core.credential.KeyCredential;
+import com.azure.core.credential.TokenCredential;
+import java.net.MalformedURLException;
+import java.net.URL;
```

Comment:
Hmmm, something seems off with formatting. I'll take a look into why this is happening

Reply:
Seems just the order of the "import".

I guess part of it is via partial-update, when java-parser merge the imports from existing file and imports from generate file.

The part on formatting seems that it no longer order the imports.